src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -176,8 +224,8 @@ protected override void Append(StringBuilder builder, LogEventInfo logEvent)
         /// Appends the Message of an Exception to the specified <see cref="StringBuilder" />.
         /// </summary>
         /// <param name="sb">The <see cref="StringBuilder"/> to append the rendered data to.</param>
-        /// <param name="ex">The exception containing the Message to append.</param>
-        protected virtual void AppendMessage(StringBuilder sb, Exception ex)
+        /// <param name="ex">The exception containing the Message to append.</param>        
+        private static void AppendMessage(StringBuilder sb, Exception ex)
```

Comment:
I get your point. With the virtual protected method (e.g. AppendMessage), you can do two things:

1. replace behavior(virtual).
2. call the method myself. This is the private/protected. 

I don't know how an `Action` will work with `virtual`, but I think this doens't work? So 
1 and 2 are now breaking changes. 2 is easy to fix, but 1 is tricky. I think you warned me before about the breaking change of 1, but I missed to see it :|

Any thoughts? Breaking changes aren't nice.